### PR TITLE
Community page

### PR DESCRIPTION
**Pull request by ekwkk**

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.UUID;
+import java.util.Set;
+import java.util.HashSet;
```
Comment (sbahaud) [inline src/main/java/com/google/codeu/data/Datastore.java:31]: Make sure the import are in alphabetic order - from the java style guide (https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -115,4 +117,19 @@ public void addMessage(List<Message> messages, Entity entity, String user) {
   return messages;
  }
 
+  /** 
+   * Gets all the users
```
Comment (sbahaud) [inline src/main/java/com/google/codeu/data/Datastore.java:121]: End the comment in a period as though it is a full sentence
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment

**Review by sbahaud — COMMENTED** [review]
It would have been better to split this up into multiple CLs. Maybe backend changes and then frontend changes

**Comment by ekwkk** [comment]
I see! I will keep that in mind next time :)

**Review by sbahaud — APPROVED** [review]
LGTM

**Pull request merged**